packages/recognize/src/lib/recognize.ts

```
@@ -0,0 +1,70 @@
+         */
+        await import('./recognize-sdk/index.js' as string);
+        recognizeApiInitialized = true;
+      } catch (err) {
+        console.error('error loading ping recognize', err);
+        return { error: 'Failed to load PingOne Recognize SDK' };
+      }
+
+      try {
+        let element: KeylessAuthElement | KeylessEnrollElement | null;
+
+        element = document.querySelector('kl-auth, kl-enroll');
```

ESLint

[error] 'element' is never reassigned. Use 'const' instead.

(prefer-const)